paper-input-container.html

```diff
@@ -277,6 +279,20 @@
         @apply(--paper-input-container-input);
       }
 
+	  .input-content ::content input:focus,
```

Comment:
Also this should use something about `this.focused`, so maybe `:host([focused]) .input-content ::content input`? (we use `focused` to style the floating label in the computed class)